config/samples/lws_v1alpha1_configuration.yaml

```diff
@@ -1,12 +1,33 @@
 apiVersion: lws.x-k8s.io/v1alpha1
 kind: Configuration
 metadata:
-  labels:
-    app.kubernetes.io/name: configuration
-    app.kubernetes.io/instance: configuration-sample
-    app.kubernetes.io/part-of: lws
-    app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/created-by: lws
   name: configuration-sample
 spec:
-  # TODO(user): Add fields here
+  ## Examples as below (most are default values):
```

Comment:
In my opinion, we should only have the ones that are actually the default.

Reply:
thanks @ardaguclu, althrouhg the default behaviors of `net.JoinHostPort` (pkg/webhook/server.go) will be 0.0.0.0 by default, but in lws code, the default of `webhook.host` is blank (""), will modify it. thank you .

Reply:
all fixed.